addons/rose/addon/components/rose/filter-tags/index.js

```diff
@@ -0,0 +1,81 @@
+import Component from '@glimmer/component';
+import { inject as service } from '@ember/service';
+import { action } from '@ember/object';
+
+export default class RoseFilterTagsIndexComponent extends Component {
+  // =services
+
+  @service router;
+
+  // =attributes
+  get tags() {
+    const tags = [];
+
+    for (const key in this.args.tags) {
+      if (Array.isArray(this.args.tags[key])) {
```

Comment:
Thinking about this, will the tags ever not be an array since this is strictly for filters?

Reply:
No you are right. When I was first writing this, I wasn't sure if we would have non-array filters but we won't. Removed check.